src/main/frontend/handler/export.cljs

```diff
@@ -219,3 +223,40 @@
       (.setAttribute anchor "href" data-str)
       (.setAttribute anchor "download" (file-name (str repo "_roam") :json))
       (.click anchor))))
+
+(defn backup-db-graph
+  [repo]
+  (when (and repo (= repo (state/get-current-repo)))
+    (when-let [backup-folder (ldb/get-key-value (db/get-db repo) :logseq.kv/graph-backup-folder)]
+      (p/let [handle (idb/get-item (str "file-handle/" backup-folder))
```

Comment:
Should the item namespace be consistent with nfs.cljs? It seems to be "handle/" everywhere else

Reply:
Handled by https://github.com/logseq/logseq/commit/78c19aacfdb3b219eb07856c5e1b0d45c3074885.
The handle needs to be separated from our old web demo graphs, it also need to be unique for each graph.